schema_based_on_diagram.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE patients (
+    id SERIAL PRIMARY KEY,
+    name varchar(100),
+    date_of_birth date
+);
+
+CREATE TABLE medical_histories (
+    id SERIAL PRIMARY KEY,
+    admitted_at TIMESTAMP,
+    patients_id INT,
+    status varchar(255),
+    FOREIGN KEY(patients_id) REFERENCES patients(id)
+ );
+ 
+CREATE TABLE treatments(
+    id serial PRIMARY KEY, 
+    type VARCHAR(50),
+    name VARCHAR(50)
+    );
+    
+CREATE INDEX treatment_histories ON medical_histories(id);
```

Comment:
- [ ] I have seen that you have one index created at line 21 as you @Farida-Hamid  have said. However, according to the project requirements, you are required to create indexes for all the foreign keys for all the tables in your project. What this means is that you need to have a total of six indexes, this includes the 5 primary tables plus one for the join table. I hope this is much clearer now for your team.

![image](https://user-images.githubusercontent.com/86318284/200777773-fb0d09d5-7401-4eab-a4b8-4d221aa3a668.png)